rts/System/ScopedResource.h

```diff
@@ -37,12 +37,6 @@ namespace spring {
 			if (!released) d(r);
 		}
 
-		const R&& Release() const {
```

Comment:
This one might still make sense w/o the first `const`?

Reply:
It tries to modify `released`, so unless `released` is mutable, you can't do that.